web/playground/src/generateBook.js

```diff
@@ -0,0 +1,138 @@
+const { readdir, stat, readFile, writeFile } = require("fs/promises");
```

Comment:
Wait, `generateBook.js` should not be part of `src/`, this would go into `tools/` or top-level.